scaladoc/src/dotty/tools/scaladoc/tasty/comments/MemberLookup.scala

```diff
@@ -128,27 +128,26 @@ trait MemberLookup {
       // val syms0 = syms.toList
       // val matched0 = syms0.filter(matches)
       // if matched0.isEmpty then
-      //   println(s"Failed to look up $q in $owner; all members below:")
+      //   println(s"Failed to look up $q in $owner; all members: {{{")
       //   syms0.foreach { s => println(s"\t$s") }
+      //   println("}}}")
       // val matched = matched0.iterator
 
-      // def showMatched() = matched.foreach { s =>
-      //   println(s">>> $s")
-      //   println(s">>> ${s.pos}")
-      //   println(s">>> [${s.flags.show}]")
-      //   println(s">>> {${if s.isTerm then "isterm" else ""};${if s.isType then "istype" else ""}}")
-      //   println(s">>> moduleClass = ${if hackResolveModule(s) == s then hackResolveModule(s).show else "none"}")
+      // def showMatched() = matched0.foreach { s =>
+      //   println(s"\t $s")
       // }
-      // println(s"localLookup in class ${owner} for `$q`{forceTerm=$forceTerm}")
-      // println(s"\t${matched0.mkString(", ")}")
+      // println(s"localLookup in class ${owner} for `$q`{forceTerm=$forceTerm}:")
       // showMatched()
 
       val matched = syms.filter(matches)
       matched.map(hackResolveModule)
     }
 
     if owner.isPackageDef then
-      findMatch(hackMembersOf(owner))
+      findMatch(hackMembersOf(owner) ++ hackMembersOf(owner).flatMap {
+        s =>
+          if s.name.endsWith("package$") then hackMembersOf(s) else Iterator.empty
+      })
```

Comment:
Maybe would be more concise

```suggestion
      findMatch(hackMembersOf(owner).flatMap {
        s =>
          (if s.name.endsWith("package$") then hackMembersOf(s) else Iterator.empty) ++ Iterator(s)
      })
```